src/marqo/core/distributed_lock/marqo_kazoo_client.py

```diff
@@ -0,0 +1,12 @@
+from kazoo.client import KazooClient
+
+
+class MarqoKazooClient(KazooClient):
```

Comment:
I wonder if `marqo.vespa` is a better place for this. Also you could call is `ZookeeperClient`, similar to `VespaClient`. Kazoo is a library to access Zookeepr, this is not a Kazoo client.

Reply:
I don't think it's the perfect place as the directory is named as `vespa`. But I think this is better than `core` as this client should parallel with `vespa_client`